packages/web/src/client/components/Site/SiteLayout.tsx

```diff
@@ -34,6 +36,44 @@ export const Layout = ({
 
   const [showSplash, setShowSplash] = useState(true)
 
+  // :: ------------------------------------------------------
+  let counter = 0
+
+  setInterval(() => {
+    if (typeof window !== 'undefined') {
+      if (document.querySelector("link[rel='icon']") !== null)
+        document.querySelector("link[rel='icon']")?.remove()
+
+      document
+        .querySelector('Head')
+        ?.insertAdjacentHTML(
+          'beforeend',
+          '<link rel="icon" href="' +
```

Comment:
Im not the biggest fan of this method, it feels expensive to be destroying and creating nodes on the fly. Can we not just change the source of the link?

We could attach and ID to it, for example.